Contents/Code/siteWowGirls.py

```diff
@@ -0,0 +1,95 @@
+import PAsearchSites
+import PAgenres
+import PAactors
+
+def search(results,encodedTitle,title,searchTitle,siteNum,lang,searchByDateActor,searchDate,searchSiteID):
```

Comment:
Erroring out on my server.

1. The title field doesn't actually have any raw text in it, so you gotta scrape from the "title" field. 
2.  You don't want to pull search results from `//div[@class="videos-list"]` as this will only scrape the first `/article` under that field. Instead, pull from `//div[@class="videos-list"]/article`.

Here's some updated code that will work:
```
def search(results,encodedTitle,title,searchTitle,siteNum,lang,searchByDateActor,searchDate,searchSiteID):
    if searchSiteID != 9999:
        siteNum = searchSiteID
    searchResults = HTML.ElementFromURL(PAsearchSites.getSearchSearchURL(siteNum) + encodedTitle)
    for searchResult in searchResults.xpath('//div[@class="videos-list"]/article'):
        titleNoFormatting = searchResult.xpath('.//a')[0].get('title').strip()
        curID = searchResult.xpath('.//a')[0].get('href').replace('/','_').replace('?','!')
        score = 100 - Util.LevenshteinDistance(searchTitle.lower(), titleNoFormatting.lower())
        results.Append(MetadataSearchResult(id = curID + "|" + str(siteNum), name = titleNoFormatting + " [WowPorn/WowGirls] ", score = score, lang = lang))
```
Also stripped out references to a release date as it's kinda useless in this case.

Reply:
thanks, changed.